src/router/router.ts

```diff
@@ -32,22 +30,3 @@ const redirectOldHashHistoryUrlPaths = (): NavigationGuard => (to, _from, next)
     next()
   }
 }
-
-/**
- * Redirects the user to the appropriate onboarding view if they haven’t completed it, yet.
- *
- * Redirects the user to the home view if they’re navigating to an onboarding route while having already completed onboarding. An exception is made when we suggest onboarding for users who don’t have data plane proxies, yet (we show an alert suggesting it and allow going to the onboarding again).
- */
-export const onboardingRouteGuard = (store: Store<State>): NavigationGuard => (to, _from, next) => {
-  const isOnboardingCompleted = store.state.onboarding.isCompleted
-  const isOnboardingRoute = to.meta.onboardingProcess
-  const shouldSuggestOnboarding = store.getters.shouldShowOnboardingNotification
-
-  if (isOnboardingCompleted && isOnboardingRoute && !shouldSuggestOnboarding) {
-    next({ name: 'home' })
```

Comment:
**Issue**: There is a regression here. Having completed onboarding no longer redirects to "home" when trying to access an onboarding route.

Reply:
Having looked at this in quite a bit of detail, I really don't think we should have this redirect, in fact as it stands before this PR its a little buggy:

1. I think the intention here was to load the onboarding the first time you hit the application. But it looks like sometimes the information needed to know whether we should show you the onboarding or not hasn't loaded yet, this means we don't then show the onboarding as you hit the application. Then unfortunately when you then click on any link on the application, the data by this time has loaded, so instead of taking you to where you wanted to go, we throw you into the onboarding screen.
2. Previous to this PR, even if I had completed onboarding, you could still visit the onboarding by clicking the 'Get started' button in the onboarding notification. The thing is, if I then happened to refresh my page whilst going through the onboarding, it would throw me back out to the home page again.

There are a few more edge cases I can think of where the previous behaviour is at least problematic, and I think this is a case of us trying to do too much, and considering that even if we did want to do this, the value it adds is very very small.

Because of this I have deleted the route guard entirely. If you now go to one of the onboarding screens again by maybe using a link I had saved previously, or clicking on a link shared by somebody else, then I won't be unexpectedly redirected to the home page.

All in all, I actually think the behaviour without the route guard is an improvement.